tests/common/data/reference_graphs/passes/dropout_synthetic_model.dot

```diff
@@ -0,0 +1,19 @@
+strict digraph  {
```

Comment:
Where is the original `.dot` for comparison of the test correctness?

Reply:
The target NNCFGraph is built in test code, should we really check its dot representation? Should we add graph test on test NNCFGraph? Or is it only for debugging?

Reply:
IMO. The code representation without visual reference takes some time to imagine the graph that exists before the pass was applied. In the case of huge and complicated graphs, it would be very inappropriate not to provide such `.dot`s.
We have `original_nncf_graph` sub-folders for 3/4 backends for some reason, don't we?